pom.xml

```diff
@@ -362,7 +362,7 @@
             <dependency>
                 <groupId>org.spdx</groupId>
                 <artifactId>tools-java</artifactId>
-                <version>1.0.4</version>
+                <version>1.1.0</version>
```

Comment:
Please add the version on the same way as the other dependency version on top of the file